website/docs/5.0-upgrade-guide.html.markdown

```diff
@@ -72,6 +72,10 @@ Please follow the format in the example below for listing breaking changes in re
 * The deprecated `storage_account_name` property has been removed in favour of the `storage_account_id` property.
 * The deprecated `resource_manager_id` property has been removed in favour of the `id` property.
 
+### `azurerm_sentinel_alert_rule_fusion`
+
+* The deprecated `name` property has been removed.
+
```

Comment:
As specified in [point 4 for breaking schema changes](https://github.com/hashicorp/terraform-provider-azurerm/blob/main/contributing/topics/guide-breaking-changes.md#breaking-schema-changes-and-deprecations) these should be listed in alphabetical order, so should be put before all the storage changes above